src/BloomExe/Browser.cs

```diff
@@ -353,6 +358,11 @@ void OnDomKeyPress(object sender, DomKeyEventArgs e)
 					PasteFilteredText(false);
 				}
 			}
+			if (Palaso.PlatformUtilities.Platform.IsMono && _controlKeyEvent != null && e.CtrlKey && e.KeyChar == 'n')
```

Comment:
How about a comment here like:

On Windows, windows forms ProcessCmdKey (intercepted in Shell) seems to get ctrl messages even when the browser has focus. But on mono, it doesn't. So we just do the same thing as that Shell.ProcessCmdKey function does, which is to raise this event.